packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -183,6 +183,11 @@ module.exports = {
               },
             ],
           },
+          // The GraphQL loader preprocesses GraphQL queries in .graphql files.
+          {
+            test: /\.(graphql)$/,
+            loader: 'graphql-tag/loader',
```

Comment:
Is there any particular reason that this loader is only registered in `dev`, but not in `prod` mode? (See the `webpack.config.prod.js`)